bundle/Resources/init_ini/siteaccess/legacy_admin/site.ini.append.php

```diff
@@ -0,0 +1,33 @@
+<?php /* #?ini charset="utf-8"?
+
+[SiteSettings]
+DefaultPage=content/dashboard
+LoginPage=custom
+
+[SiteAccessSettings]
+RequireUserLogin=true
+RelatedSiteAccessList[]
+RelatedSiteAccessList[]=site
+#RelatedSiteAccessList[]=admin
```

Comment:
Why commented out?

Reply:
admin is not a legacy siteaccess, so I was unsure if it should be here or not. Probably not.

Reply:
We always kept it, didn't matter. Probably doesn't matter either way.

Reply:
well, it's used misc places to provide list of siteaccess to see things, like preview, template view and ini view. Which is why I think removing it might be best.

Reply:
Removed it given it was not present in override settings.